### PR TITLE
VS-43: Nuget release automation

### PR DESCRIPTION
**Pull request by BorisDog**
[Sample myget run](https://spruce.mongodb.com/version/663e6c6a92d24300072b6d5e)

### CODE REVIEW

File: evergreen/run-pack.sh [hunk 1]
```diff
@@ -1,11 +1,6 @@
 #!/usr/bin/env bash
-set -o errexit  # Exit the script with error if any of the commands fail
-
-if [ -z "$PACKAGE_VERSION" ]
-then
-  PACKAGE_VERSION=$(git describe --tags)
-  echo Calculated PACKAGE_VERSION value: "$PACKAGE_VERSION"
-fi
+set -o errexit # Exit the script with error if any of the commands fail
+set +o xtrace  # Disable tracing.
```
Comment (sanych-sun) [inline evergreen/run-pack.sh:4]: Should we validate if the parameters were set?

File: src/MongoDB.Analyzer.Package/MongoDB.Analyzer.Package.csproj [hunk 2]
```diff
@@ -10,14 +10,14 @@
 
   <PropertyGroup>
     <PackageId>MongoDB.Analyzer</PackageId>
-    <PackageVersion>1.3.0</PackageVersion>
+    <PackageVersion>$(Version)</PackageVersion>
```
Comment (sanych-sun) [inline src/MongoDB.Analyzer.Package/MongoDB.Analyzer.Package.csproj:13]: Do we need this here? As far as I understood the package version is set by dotnet cli parameters.

**Review by sanych-sun — CHANGES_REQUESTED** [review]
Looks good, minor changes requested.

File: evergreen/run-pack.sh (hunk 1 above)
Comment (BorisDog) [reply evergreen/run-pack.sh:4]: Added validation.

File: src/MongoDB.Analyzer.Package/MongoDB.Analyzer.Package.csproj (hunk 2 above)
Comment (BorisDog) [reply src/MongoDB.Analyzer.Package/MongoDB.Analyzer.Package.csproj:13]: Leaving this for better readability (value in `PackageVersion` overrides the passed in cli)

**Review by sanych-sun — APPROVED** [review]
LGTM

**Pull request merged**